### ISSUE TITLE
Model Evaluation and optimisation

### ISSUE THREAD

**Issue by MeloR1** [Seravian/Seravian-AI#3]
Training loss is higher than expected (9.5). Find a way to reduce the training loss using AdamW optimizers or any other way
